pkg/loghttp/query.go

```diff
@@ -441,3 +561,7 @@ func labelVolumeLimit(r *http.Request) error {
 
 	return nil
 }
+
+func yoloString(buf []byte) string {
+	return *((*string)(unsafe.Pointer(&buf)))
+}
```

Comment:
the scariest thing in the PR ) we should use it only if we check that the buffer is not updated, however, a lot of functions in stack trace mean that somebody can do some optimization to reuse the slice and it will break all our implementation. Why do we need it here?